php/libraries/LorisForm.class.inc

```diff
@@ -677,14 +677,26 @@ class LorisForm
     protected function selectHTML($el)
     {
         $strOptions    = '';
-        $options       = $el['options'];
+        $options       = $el['options'] ?? array();
         $cls           = '';
         $changehandler = '';
         $multiple      = '';
         $id            = '';
         $disabled      = '';
-        $name          = $el['name'];
-
+        $name          = $el['name'] ?? '';
+
+        if (!is_array($options) || !($options instanceof Traversable)) {
```

Comment:
* [Solution from here](https://stackoverflow.com/questions/2630013/invalid-argument-supplied-for-foreach)

* [Array does not implement Traversable](https://secure.php.net/manual/en/class.traversable.php#118575)

- [x] Next commit fixes boolean logic